tests/ha_proxy_server_test.go

```diff
@@ -136,6 +138,12 @@ func TestBasicHAProxyServer_GRPC(t *testing.T) {
 		}
 	}
 	if !ready {
+		t.Logf("Proxy-0(%s<->%s), Proxy-1(%s<->%s), Proxy-2(%s<->%s)",
```

Comment:
Are these also leftover of debugging code? There are a few other places adding t.Logf, can you double check if they are needed?

Reply:
Fixed